diff --git a/common/public/property.te b/common/public/property.te
index 17c5cd0..f31c7b3 100644
--- a/common/public/property.te
+++ b/common/public/property.te
@@ -1,5 +1,4 @@
 # Aux camera allow/excludelist prop
-system_vendor_config_prop(vendor_persist_camera_prop)
 
 # NFC
 system_vendor_config_prop(vendor_persist_nfc_prop)
diff --git a/qcom/vendor/fm_app.te b/qcom/vendor/fm_app.te
index c37e8f5..e69de29 100644
--- a/qcom/vendor/fm_app.te
+++ b/qcom/vendor/fm_app.te
@@ -1 +0,0 @@
-get_prop(vendor_fm_app, vendor_fm_radio_app_prop)
diff --git a/qcom/vendor/hal_gnss_qti.te b/qcom/vendor/hal_gnss_qti.te
index 4e914a2..463d341 100644
--- a/qcom/vendor/hal_gnss_qti.te
+++ b/qcom/vendor/hal_gnss_qti.te
@@ -1,2 +1 @@
 # xtra-daemon control
-dontaudit hal_gnss_qti xtra_control_prop:file r_file_perms;
diff --git a/qcom/vendor/hal_lineage_health_default.te b/qcom/vendor/hal_lineage_health_default.te
index 26de236..eb922f9 100644
--- a/qcom/vendor/hal_lineage_health_default.te
+++ b/qcom/vendor/hal_lineage_health_default.te
@@ -1,2 +1,2 @@
-rw_dir_file(hal_lineage_health_default, sysfs_battery_supply)
-rw_dir_file(hal_lineage_health_default, sysfs_usb_supply)
+allow hal_lineage_health_default vendor_sysfs_battery_supply:dir r_dir_perms;
+allow hal_lineage_health_default vendor_sysfs_battery_supply:file rw_file_perms;
diff --git a/qcom/vendor/hal_lineage_livedisplay_qti.te b/qcom/vendor/hal_lineage_livedisplay_qti.te
index 4950726..aaf35d2 100644
--- a/qcom/vendor/hal_lineage_livedisplay_qti.te
+++ b/qcom/vendor/hal_lineage_livedisplay_qti.te
@@ -1,14 +1,3 @@
 type hal_lineage_livedisplay_qti, domain;
-hal_server_domain(hal_lineage_livedisplay_qti, hal_lineage_livedisplay)
-
 type hal_lineage_livedisplay_qti_exec, exec_type, vendor_file_type, file_type;
 init_daemon_domain(hal_lineage_livedisplay_qti)
-
-binder_call(hal_lineage_livedisplay_qti, servicemanager)
-
-# Allow LiveDisplay HAL's default implementation to use vendor-binder service
-vndbinder_use(hal_lineage_livedisplay_qti)
-
-# Allow LiveDisplay to store files under /data/vendor/display and access them
-allow hal_lineage_livedisplay_qti display_vendor_data_file:dir rw_dir_perms;
-allow hal_lineage_livedisplay_qti display_vendor_data_file:file create_file_perms;
diff --git a/qcom/vendor/hal_perf_default.te b/qcom/vendor/hal_perf_default.te
index 9125357..fb932ce 100644
--- a/qcom/vendor/hal_perf_default.te
+++ b/qcom/vendor/hal_perf_default.te
@@ -1,3 +1,4 @@
-r_dir_file(hal_perf_default, hal_power_default)
-
-rw_dir_file(hal_perf_default, proc_sched)
+allow hal_perf_default hal_power_default:dir r_dir_perms;
+allow hal_perf_default hal_power_default:file r_file_perms;
+allow hal_perf_default proc_sched:dir r_dir_perms;
+allow hal_perf_default proc_sched:file rw_file_perms;
diff --git a/qcom/vendor/hal_power_default.te b/qcom/vendor/hal_power_default.te
index 969819c..7328cba 100644
--- a/qcom/vendor/hal_power_default.te
+++ b/qcom/vendor/hal_power_default.te
@@ -1,6 +1,10 @@
-# To do powerhint on nodes defined in powerhint.json
-rw_dir_file(hal_power_default, proc_sched)
-rw_dir_file(hal_power_default, sysfs_devfreq)
-rw_dir_file(hal_power_default, sysfs_graphics)
-rw_dir_file(hal_power_default, sysfs_kgsl)
-rw_dir_file(hal_power_default, sysfs_scsi_host)
+allow hal_power_default proc_sched:dir r_dir_perms;
+allow hal_power_default proc_sched:file rw_file_perms;
+allow hal_power_default sysfs_devfreq:dir r_dir_perms;
+allow hal_power_default sysfs_devfreq:file rw_file_perms;
+allow hal_power_default sysfs_graphics:dir r_dir_perms;
+allow hal_power_default sysfs_graphics:file rw_file_perms;
+allow hal_power_default sysfs_kgsl:dir r_dir_perms;
+allow hal_power_default sysfs_kgsl:file rw_file_perms;
+allow hal_power_default sysfs_scsi_host:dir r_dir_perms;
+allow hal_power_default sysfs_scsi_host:file rw_file_perms;
diff --git a/qcom/vendor/location.te b/qcom/vendor/location.te
index fcb1a48..9998132 100644
--- a/qcom/vendor/location.te
+++ b/qcom/vendor/location.te
@@ -1,4 +1,3 @@
 # xtra-daemon control
-get_prop(location_domain, xtra_control_prop)
 
 dontaudit location_domain sysfs_socinfo_sensitive:file r_file_perms;
